templates/company-board/test/company-board.js

```diff
@@ -43,59 +53,111 @@ contract('Company with board', ([_, owner, boardMember1, boardMember2, shareHold
+          DEFAULT_FINANCE_PERIOD
+        ), 'COMPANY_BAD_HOLDERS_STAKES_LEN')
       })
     })
   })
 
   context('when the creation succeeds', () => {
     before('create company entity', async () => {
-      prepareReceipt = await template.prepareInstance({ from: owner })
-      setupReceipt = await template.setupInstance(daoID, BOARD_MEMBERS, SHARE_HOLDERS, SHARE_STAKES, { from: owner })
+      prepareReceipt = await template.prepareInstance(SHARE_TOKEN_NAME, SHARE_TOKEN_SYMBOL, { from: owner })
+      setupReceipt = await template.setupInstance(
+        daoID, 
+        BOARD_MEMBERS, 
+        SHARE_HOLDERS, 
+        SHARE_STAKES, 
+        [BOARD_VOTE_DURATION, BOARD_SUPPORT_REQUIRED, BOARD_MIN_ACCEPTANCE_QUORUM],
+        [SHARE_VOTE_DURATION, SHARE_SUPPORT_REQUIRED, SHARE_MIN_ACCEPTANCE_QUORUM],
+        DEFAULT_FINANCE_PERIOD,
+        { from: owner }
+      )
 
       dao = Kernel.at(getEventArgument(prepareReceipt, 'DeployDao', 'dao'))
       boardToken = MiniMeToken.at(getEventArgument(prepareReceipt, 'DeployToken', 'token', 0))
       shareToken = MiniMeToken.at(getEventArgument(prepareReceipt, 'DeployToken', 'token', 1))
     })
 
     before('load apps', async () => {
-      const installedApps = getInstalledAppsById(setupReceipt)
+      const installedApps = getInstalledAppsById(setupReceipt);
       assert.equal(installedApps.agent.length, 1, 'should have installed 1 agent app')
-      assert.equal(installedApps.voting.length, 2, 'should have installed 2 voting apps')
       assert.equal(installedApps.finance.length, 1, 'should have installed 1 finance app')
+      assert.equal(installedApps.voting.length, 2, 'should have installed 2 voting apps')
       assert.equal(installedApps['token-manager'].length, 2, 'should have installed 2 token manager apps')
-
       acl = ACL.at(await dao.acl())
       agent = Agent.at(installedApps.agent[0])
+      finance = Finance.at(installedApps.finance[0])
       boardVoting = Voting.at(installedApps.voting[0])
       shareVoting = Voting.at(installedApps.voting[1])
-      finance = Finance.at(installedApps.finance[0])
       boardTokenManager = TokenManager.at(installedApps['token-manager'][0])
       shareTokenManager = TokenManager.at(installedApps['token-manager'][1])
     })
 
     it('costs ~10.4e6 gas', async () => {
-      assert.isAtMost(prepareReceipt.receipt.gasUsed, 5e6, 'prepare script should cost almost 5e6 gas')
-      assert.isAtMost(setupReceipt.receipt.gasUsed, 5.4e6, 'setup script should cost almost 5.4e6 gas')
+      const prepareGas = prepareReceipt.receipt.gasUsed;
+      const setupGas = setupReceipt.receipt.gasUsed;
+      const totalGas = prepareGas + setupGas;
+      assert.isAtMost(prepareGas, 5.0e6, 'prepare script should cost almost 5.0e6 gas')
+      assert.isAtMost(setupGas, 5.4e6, 'setup script should cost almost 5.4e6 gas')
+      assert.isAtMost(totalGas, 10.4e6, 'prepare + setup scripts should cost almost 10.4e6 gas');
```

Comment:
Note on the trailing `;` too, generally we don't use them anymore :)

Reply:
:+1: 
I'll also be setting up my linter for JS this week to comply with the team's JS coding style.